app/helpers/global_stats_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GlobalStatsHelper
```

Comment:
Do we need this file? It seems that since this is being used as an API endpoint and there are not any views utilizing this there is no reason for this `ActionView::Helper` module.

Reply:
Nope. Removed the file. :+1: